### ISSUE TITLE
Pane split vertically is different on non-English keyboard layouts

### ISSUE THREAD

**Issue by pellelauritsen** [microsoft/terminal#3737]
I'm using an English/US Windows 10 Pro 18362 with Danish keyboard layout. And the short cut for pane split vertically is Alt + Shift + "+"

**Comment by enricoreich** [comment]
Same here with German/DE keybord layout.

**Comment by VikingScientist** [comment]
Norwegian keyboard and I observe the same thing. I assume it is intentional as the = key is already hidden behind a shift modifier. Anyway, I'm mostly happy with the current solution so I only think the documentation needs an update.

My only complaint is that multi-language keyboard setups [use shift+alt](https://support.office.com/en-us/article/switch-between-languages-using-the-language-bar-1c2242c0-fe15-4bc3-99bc-535de6f4f258) as the hotkey to change between english and native keyboard layouts. This causes some confusion when combined with the fact that alt+shift is heavily used in pane control. Note that "split vertically" (shift+alt+ '+') does *not* change layout, but "I think I'm going to add another tab, but on second thought I had a change of heart" (shift+alt), *will* change your keyboard layout; and consequently move the new pane button to a different place on your keyboard.

**Comment by hanubeki** [comment]
I saw the line in defaults.json:

```json
{ "command": "splitVertical", "keys": [ "alt+shift+plus" ] },
```

So I think the release note for v0.7.3291.0 is not correct.

**Comment by DHowett-MSFT** [comment]
>I assume it is intentional as the = key is already hidden behind a shift modifier.

Yep, that was me simplifying the release notes based on my US-104 keyboard. The true binding is `alt+shift+plus`, which can be expressed differently depending on your keyboard.

For a US-104 keyboard, `alt+plus` and `alt+shift+=` are synonymous with `alt+shift+plus`. Unfortunately, that doesn't hold for everyone else.

I'm going to file a followup issue for the <kbd>Shift+Alt</kbd> thing and close this one out when the release notes get updated.
